toolkit/primitives/domain/src/lib.rs

```diff
@@ -300,9 +300,12 @@ impl MainchainKeyHash {
 	}
 }
 
+/// ECDSA signature of MainchainPrivateKey, 64 bytes.
+const MAINCHAIN_SIGNATURE_LEN: usize = MAINCHAIN_PRIVATE_KEY_LEN * 2;
```

Comment:
can we remove `pub` attribute from `MAINCHAIN_KEY_HASH_LEN` in line 277 once we are here?